server/plugin/client.go

```diff
@@ -161,8 +184,13 @@ func (c *client) CreateSubscription(body *serializers.CreateSubscriptionRequestP
 		},
 	}
 
+	baseURL := c.plugin.getConfiguration().AzureDevopsAPIBaseURL
+	if strings.Contains(body.EventType, "release") {
+		baseURL = strings.Replace(baseURL, "://", "://vsrm.", 1)
+	}
```

Comment:
create an util for this as it will be used frequently, and for now you can add a TODO comment on the util to use that at all such places where original baseURL is to be updated this way